readme.md

```diff
@@ -63,11 +64,11 @@ Check to make sure you have Ruby installed: `$ ruby -v`
 $ touch app.rb        # Create a Ruby file
 $ ruby file_name.rb   # Run that ruby file
 ```
-> This is the equivalent of running `node script.js` in the Terminal.
+> This is the equivalent of running `$ node script.js` in the Terminal
 
-#### PRY (REPL)
+#### By REPL
 
-Just like Javascript and the browser console, we can use tools like Pry and IRB to run Ruby in a REPL.
+Just like Javascript and the browser console, we can use tools like Pry and IRB to run Ruby in a REPL
```

Comment:
Just like how used the browser console to test Javascript code, we can use tools like Pry and IRB to run Ruby in a REPL.